src/lib/capabilities.tsx

```diff
@@ -142,7 +144,7 @@ export const getCapabilities = (
     formatArticle: !isViewer(),
     /* editor */
     editArticle: !isViewer(),
-    editMetadata: !(isViewer() || isAnnotator()),
+    editMetadata: !(isViewer() || isProofer()),
```

Comment:
why remove Annotater, as he can't change any model. so now the backend will allow it!

Reply:
@asouqi this is was requested on https://jira.atypon.com/browse/LEAN-2584

-The Annotator role should have access to the "edit authors" button

The Annotator role should be able to click on the Authors' links to edit the details

Reply:
if they didn't change that in the backend that will not work, annotator can't change MPModel